src/test/java/com/openshift/internal/restclient/model/build/WebhookTriggerTest.java

```diff
@@ -29,13 +29,13 @@ public void setup(){
 	}
 	@Test
 	public void testGetWebhookUrlWhenResourceHasBaseURL() {
-		trigger = new WebhookTrigger(BuildTriggerType.generic, "secret101","foo","https://localhost:8443","v1beta1","test");
-		assertEquals("https://localhost:8443/osapi/v1beta1/buildConfigHooks/foo/secret101/generic?namespace=test", trigger.getWebhookURL());
+		trigger = new WebhookTrigger(BuildTriggerType.GENERIC, "secret101","foo","https://localhost:8443","v1beta1","test");
+		assertEquals("https://localhost:8443/osapi/v1beta1/buildConfigHooks/foo/secret101/Generic?namespace=test", trigger.getWebhookURL());
```

Comment:
I just came across this when I was replacing the deprecated enum in the test and noticed the test failing because of the changed casing. Was thus wondering if the result was right.